src/test/java/JunitTest.java

```diff
@@ -1,9 +1,38 @@
 import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
 
 public class JunitTest {
     @Test
-    public void testIsNumberPositive() {
-        StringUtils stringUtils = new StringUtils();
-        assert stringUtils.isPositiveNumber("1");
+    public void testIsNumberPositive() throws Exception {
```

Comment:
Great approach to test your code. To make it better:
1. lets rename the class to tell what class it tests eg. TestStringUtils or TestStringUtilsPositiveNumber, feel free to name it your own way, but remember to give appropriate and explicable name
2. lets break down test cases into single assert statement per test case.
eg. testPositiveNumber() {assertTrue(StringUtils.isPositiveNumber("1"))}
testNegativeNumber() {assertFalse(StringUtils.isPositiveNumber("-1"))}
Also it would be nice to check input like this: 12.234, 123.45.6, -12.345